### DISCUSSION TITLE
Total Yield

### DISCUSSION THREAD

**Discussion by MaxiKing007** [tbnobody/OpenDTU#227] [Q&A]
Hi,
hat jemand die openDTU an einem neuen Hoymiles angeschlossen? War da YieldTotal auch 0 kWh? Unser HM-400 läuft seit Anfang August und seitdem wurden 112 kWh produziert. Mit Shelly Plug S habe ich seit Anfang September nur rund 60 kWh gemessen...würde mich interessieren wann der YieldTotal Wert auf 0 gesetzt wird :)

**Comment by turrican944** [comment]
Moin
Ich habe einen HM600 seit ein paar Tagen an laufen und der hat jetzt knapp 3 kWh im OpenDTU. War also wohl 0 und auch gerade wenig Sonne. Habe den aber zuerst ohne OpenDTU betrieben, also ob wirklich 0 drin stand kann ich nicht sagen.

**Comment by dersch81** [comment]
Ich habe einen HM350 seit Juli im Einsatz und einen ShellyPM zum messen dran. Nun habe ich OpenDTU im Einsatz und YieldTotal deckt sich mit dem bisherigen Ertrag.

Nun habe ich noch 2 weitere, neue Hoymiles installiert (HM600 und HM300) und die haben bei 0 angefangen.

**Reply by MaxiKing007** [reply]
Ok cool dann ist die Shelly Plug S ungenau bzw. Misst zu wenig und wir haben wirklich schon 111kWh produziert

**Reply by MaxiKing007** [reply]
Komisch dass die Shelly PM das gleiche wie der Hoymiles misst? Die Shelly misst doch auch AC-seitig...

**Reply by dersch81** [reply]
Die Werte waren nicht zu 100% gleich daher passt das schon.

**Reply by wuppiwuppi** [reply]
Mein Shelly PM war murks, hat ca 10% mehr angezeigt. Der Shelly EM ist ziemlich genau, aber dank dieses Projektes ist der überflüssig.

**Comment by ghost** [comment]
Unser HM-600 hat bei 0 kWh angefangen zu zählen. Bei Vergleichen mit anderen Stromzählern ist zu berücksichtigen, dass die "YieldTotal" des HM-600 die kumulierten **DC**-kWh sind; bei den (eingespeisten) **AC**-kWh ist der Wirkungsgrad des WR zu berücksichtigen. Bei einem Shelly EM (mit Stromwandler passen diese Werte sehr gut zusammen.
![DTU_AC_DC](https://user-images.githubusercontent.com/107404965/196640760-0bd0b553-2389-43e2-b447-6db8d0e568ca.png)

**Reply by dersch81** [reply]
Oh, das ist mal eine wichtige Info! Wenn das nicht beachtet wird verfälscht es die Statistik.

**Reply by MaxiKing007** [reply]
Interessant! Hast du es auch mit Shelly Plug S verglichen?

**Reply by ghost** [reply]
Nein. Zum Messen der kWh habe ich (nur) den schon erwähnten Shelly EM (AC-Einspeisung) im Einsatz.

**Reply by MaxiKing007** [reply]
Ok. Ein Vergleich wäre echt interessant :D aber wenn man einfach den Wirkungsgrad vom WR berücksichtigt und dann die Total Yield Werte anpasst, scheint es ja zu stimmen :) dann ist Shelly Plug S die Untergrenze und TotalYield (da DC) die Obergrenze

**Reply by MaxiKing007** [reply]
Sind die YieldDay Werte auch DC Werte?

**Reply by ghost** [reply]
Ja. Das sieht man bei Wechselrichtern mit mehr als einem String, z. B. HM-600: YieldDay (Phase 1) ist die Summe YieldDay von String 1 und String 2. String 1/String 2 sind nur DC-Werte. (Eine vergleichbare Betrachtung gilt sinngemäß für YieldTotal).
